docs/en/sql-reference/statements/show.md

```diff
@@ -428,4 +428,71 @@ errors_count:            0
 estimated_recovery_time: 0
 ```
 
-[Original article](https://clickhouse.tech/docs/en/query_language/show/) <!--hide-->
+## SHOW SETTINGS {#show-settings}
+
+Returns a list of system settings and their values. Selects data from the [system.settings](../../operations/system-tables/settings.md) table.
+
+**Syntax**
+
+```sql
+SHOW [CHANGED] SETTINGS LIKE|ILIKE <name>
+```
+
+**Clauses**
+
+`LIKE` clause needs a setting name or part of the name without globs.
```

Comment:
The only difference between like and `LIKE` and `ILIKE` is case insensitiveness (ILIKE == case insensitive like). So `%` and `_` are also allowed. And I think we should give any details about `LIKE|ILIKE` hire, just something common: 
"`LIKE|ILIKE` allows to specify a matching pattern for setting name".